book/src/dev/release-process.md

```diff
@@ -0,0 +1,118 @@
+# Zebra versioning and releases
+
+This document contains the practices that we follow to provide you with a leading-edge application, balanced and with stability.
+We strive to ensure that future changes are always introduced in a predictable way.
+We want everyone who depends on Zebra to know when and how new features are added, and to be well-prepared when obsolete ones are removed.
+
+<a id="versioning"></a>
+
+## Zebra versioning
+
+Zebra version numbers indicate the level of changes that are introduced by the release.
+This use of [semantic versioning](https://semver.org/ "Semantic Versioning Specification") helps you understand the potential impact of updating to a new version.
+
+Zebra version numbers have three parts: `major.minor.patch`.
+For example, version `3.1.11` indicates major version 3, minor version 1, and patch level 11.
+
+The version number is incremented based on the level of change included in the release.
+
+<div class="alert pre-release">
+
+**NOTE**: <br />
+As Zebra is in a `pre-release` state (is unstable and might not satisfy the intended compatibility requirements as denoted by its associated normal version).
+The pre-release version is denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version.
+
+</div>
+
+| Level of change | Details |
+|:---             |:---     |
+| Major release   | Contains significant new features, and commonly correspond to network upgrades; some technical assistance may be needed during the update. When updating to a major release, you may need to follow the specific upgrade instructions provided in the release notes. |
+| Minor release   | Contains new smaller features. Minor releases are should be fully backward-compatible. No technical assistance is expected during update.  If you want to use the new features in a minor release, you might need to follow the instructions in the release notes. |
```

Comment:
Extra "are"; should read: "Minor releases should be fully backward-compatible." instead